README.md

```diff
@@ -35,7 +31,7 @@ the [Google Gmail API](https://developers.google.com/gmail/api/).
 
 ### Running
 
-`bundle exec rake run` for the drug and medical device alert checker
+`bundle exec rake run_medical_alerts` for the medical safety alert checker
```

Comment:
Minor: this commit comes later in the sequence.

Reply:
Good spot, I've sorted this out.